docs/applications/sonarrv3.mdx

```diff
@@ -118,6 +118,34 @@ API: <copy and paste from jackett UI>
 ```
 4. Click `Test`. If all is good, click `Save`.
 
+### Add an indexer with NZBHydra
+Sonarr only supports a few indexers out of the box, but the indexers can be expanded by using the application [NZBHydra](nzbhydra.mdx). To add an indexer with NZBHydra, use the following steps.
+
+:::note
+By default, when receiving the newznab feeds from NZBHydra, it will need to be modified to be connected with the application. The default link looks like:
+
+```plaintext
+https://<yourhostname.ltd>/nzbhydra/api
```

Comment:
would `localhost` not work in these instances?

Reply:
my bad. I copied the jackett instructions. I will update.

Reply:
```suggestion
https://localhost/nzbhydra/api
```

Reply:
this snuck back in ha